frontend/src/state/backendApi.ts

```diff
@@ -253,8 +263,14 @@ const apiStore = {
                 case 'message':
                     let m = msg.message as TopicMessage;
 
-                    if (m.key && typeof m.key === 'string' && m.key.length > 0)
-                        m.key = atob(m.key); // unpack base64 encoded key
+                    if (m.key && typeof m.key === 'string' && m.key.length > 0) {
+                        try {
+                            m.key = atob(m.key); // unpack base64 encoded key
+                        } catch (error) {
+                            // Empty
+                            // Only unpack if the key is base64 based
```

Comment:
did that happen for you? 
in any case, good that you added a catch handler 👍

Reply:
Yes, it happened to some certain topics in my case